src/api/doAttachApp.js

```diff
@@ -0,0 +1,137 @@
+import { addedDiff, updatedDiff } from 'deep-object-diff';
+import { patchSource, handleError } from '@redhat-cloud-services/frontend-components-sources';
+import isEmpty from 'lodash/isEmpty';
+import merge from 'lodash/merge';
+import cloneDeep from 'lodash/cloneDeep';
+
+import { getSourcesApi, doCreateApplication } from './entities';
+import { urlOrHost } from './doUpdateSource';
+
+export const convertToUndefined = (value) => {
```

Comment:
This might not be necessary after : https://github.com/data-driven-forms/react-forms/pull/273
I think it can be merged soon

Reply:
Yeah, I am planning to use it in the future, but right now I would rather move this forward. :incoming_envelope: